### DISCUSSION TITLE
Extraneous spaces between some words e.g. FA Qs

### DISCUSSION THREAD

**Discussion by lack-of-gravitas** [directus/directus#13764] [Archive]
### Describe the Bug

Extra space added between words in some collections. Currently only observed for "FAQs" and only on Directus UI in Data Model > detail view (database table is correctly worded).

![image](https://user-images.githubusercontent.com/10094964/172271436-935d26bc-6916-46c8-9055-8d013bf74bb7.png)

![image](https://user-images.githubusercontent.com/10094964/172271263-a6f93043-b1f2-4161-836a-9caa71af8e2f.png)

![image](https://user-images.githubusercontent.com/10094964/172271391-cb12423e-a671-471b-b5d9-e723d33ea60e.png)

![image](https://user-images.githubusercontent.com/10094964/172271565-bd61b68c-224d-45ed-b461-86fe095bf334.png)

### To Reproduce

1 - create a collection and name it "ProductFAQs" (currently only known string that causes this issue)
3 - Expected Result in Data Model > detail view = **Product FAQs**
4 - Actual Result = **Product Fa Qs** (in data model detail)

### Errors Shown

No

### What version of Directus are you using?

9.12.1

### What version of Node.js are you using?

Node LTS 16.15.0

### What database are you using?

PostgreSQL 14.1 on Supabase

### What browser are you using?

Firefox Developer Edition 102.0b3 (64-bit)

### How are you deploying Directus?

Docker on Azure App Service

**Comment by rijkvanzanten** [comment]
Directus will try to auto-generate a human readable name based on the collection key if you don't have any custom translations set. In this particular case, `ProductFAQs` is treated as camelCase, and is turned into `Product Fa Qs`. You can set a custom translated name from the collection/field settings 👍🏻

**Reply by lack-of-gravitas** [reply]
figured that was the case (no pun intended)... thanks!
